test/references/Async Ecology.txt

```diff
@@ -0,0 +1,18 @@
+register(DataDep(
```

Comment:
YESSS! 
AWESOME!

I am looking at this compared to 
https://github.com/oxinabox/DataDepsGenerators.jl/blob/master/test/references/DataCite%20Ecology.txt

and

https://github.com/oxinabox/DataDepsGenerators.jl/blob/master/test/references/DataDryad%20Ecology.txt

and it has the best parts of  each!